test/util/test_tools.cpp

```diff
@@ -147,3 +147,119 @@ shared_ptr<Function> make_test_graph()
 
     return f0;
 }
+
+template <>
+void init_int_tv<char>(std::shared_ptr<ngraph::runtime::Tensor> tv,
```

Comment:
Since this function does not need any ownership of `tv` pass it as a `Tensor*` instead of shared_ptr
Same comment for all init_*_tv functions and random_init

Reply:
Done.